PyInstaller/building/splash.py

```diff
@@ -389,60 +387,71 @@ def _resize_image(_image, _orig_size):
                 img.save(image_data, format='PNG')
                 img.close()
                 image = image_data.getvalue()
-            logger.info("Converted image %s to PNG format" %
-                        self.image_file)
+            logger.info("Converted image %s to PNG format" % self.image_file)
         else:
-            raise ValueError("The image %s needs to be converted to a PNG"
-                             " file, but PIL.Image is not available! Either"
-                             " install the Pillow package, or use a PNG image"
-                             " for you splash screen." % self.image_file)
+            raise ValueError(
+                "The image %s needs to be converted to a PNG"
+                " file, but PIL.Image is not available! Either"
+                " install the Pillow package, or use a PNG image"
+                " for you splash screen." % self.image_file
+            )
 
         image_file.close()
 
-        res = SplashWriter(self.name,  # noqa: F841
-                           self.splash_requirements,
-                           self.tcl_lib[0],  # tcl86t.dll
-                           self.tk_lib[0],  # tk86t.dll
-                           TK_ROOTNAME,
-                           self.rundir,
-                           image,
-                           self.script)
+        SplashWriter(
+            self.name,  # noqa: F841
```

Comment:
Remove noqa; it was necessary before because SplashWriter instance was assigned to unused `res` variable. 

We probably don't need to worry about the instance immediately becoming eligible for garbage collection, as this is the last statement within that scope (method).